content/agent/how-to/run-agent-container.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Run the NGINX Agent in a container"
+weight: 300
+toc: true
+type: how-to
+product: Agent
+---
+
+## Overview
+
+This guide serves as a step-by-step guide to run NGINX Agent in a container. It covers the basic setup needed to get the NGINX Agent up and running efficiently and securely.
+
+## Before you begin
+
+Before you begin this guide ensure:
+
+{{< note >}}
+This guide uses Docker but NGINX Agent will also work with other container applications.
+{{< /note >}}
+
+- **Docker:** Ensure Docker is installed and configured on your system. [Download Docker from the official site](https://www.docker.com/products/docker-desktop/).
+- **NGINX Agent Image:** You need access to the Docker image for NGINX Agent. Find the appropriate image in your organization's registry or on Docker Hub if publicly available.
+- **NGINX Configuration File:** Prepare and validate your NGINX configuration files that the Agent will monitor.
+- **Credentials:** Acquire any necessary authentication tokens or credentials required for the NGINX Agent.
+
+1. Pull the NGINX Agent container image
+
+The NGINX Agent container image must be downloaded from a trusted source such as Docker Hub or a private container registry.
+
+Run the following command to pull the official image:
+
+```bash
+<!-- Registry HERE -->
+docker pull <Registry HERE>:latest
+```
+
+Ensure you are using the correct image version. Replace `latest` with the desired version tag if necessary.
+
+
+2. Create a configuration file
+
+1. Create a configuration file named `nginx-agent.conf` in your current directory.
+2. Populate the file with the following structure:
+
+```vim
```

Comment:
Is this appropriate for a config file? (I don't have a good alternative, ref https://github.com/jincheng9/markdown_supported_languages?tab=readme-ov-file#heres-a-full-list-of-supported-languages)

Reply:
```suggestion
```yaml
```
yaml works great for this block